pkgs/development/embedded/avrdude/libelf.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "libelf";
+  version = "0.8.13-unstable-2023-01-14";
+
+  src = fetchFromGitHub {
+    owner = "avrdudes";
+    repo = "libelf";
+    rev = "0c55bfe1d3020a20bddf6ce57c0d9d98ccb12586";
+    hash = "sha256-jz7Ef0Eg673IJVZvVNklY40s13LCuMVAc7FGrRI7scQ=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
```

Comment:
```suggestion
```
that's done in the cmake setup hook. https://github.com/NixOS/nixpkgs/blob/f59f29659e1db1130f87936ef9f2a11a71b3a99c/pkgs/by-name/cm/cmake/setup-hook.sh#L111

Reply:
Thanks, TIL.